client/src/styles/DetailView.scss

```diff
@@ -314,12 +286,6 @@
       }
     }
 
-    .card-body-links {
```

Comment:
don't know what this was really doing, but needed to get rid of it for proper padding between cards in mobile